Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -144,54 +145,60 @@ TEST(BloomFilterUnitTest,
 
 class BloomFilterGoldenTest : public ::testing::Test {
  public:
-  void RunGoldenTest(std::string& test_file) {
-    size_t start_pos = test_file.find("bloom_filter_proto");
-    if (start_pos == std::string::npos) {
-      return;
-    }
+  void RunGoldenTest(const std::string& test_file) {
+    const std::string substring = "bloom_filter_proto";
```

Comment:
I think it would help readability if the logic to calculate the result file from the test file were moved into a helper method. Could you do that and see if it improves readability in your opinion? Note that the HARD_ASSERT can be part of that helper method. Also, the helper method can take the test file string _by value_ (as opposed to _by reference_), mutate it, and return it. Then you don't need to explicitly make a copy.

Reply:
Re-arranged the code in RunGoldenTest with helper functions.